src/store/modules/search.store.js

```diff
@@ -53,12 +53,12 @@ const actions = {
             const coordinate = coordinateFromString(query)
             if (coordinate) {
                 dispatch('setCenter', coordinate)
-                dispatch('setZoom', ZOOM_LEVEL_1_25000_MAP)
+                dispatch('setZoom', MERCATOR_ZOOM_LEVEL_1_25000_MAP)
```

Comment:
Why search is setting the zoom to web mercator ? Should it no set it depending on the current projecion ?